integration_tests/commands/resp/getunwatch_test.go

```diff
@@ -8,13 +8,21 @@ import (
 	"time"
 
 	"github.com/dicedb/dice/internal/clientio"
+	"github.com/dicedb/dice/internal/cmd"
 	"github.com/dicedb/dicedb-go"
 	"github.com/stretchr/testify/assert"
 )
 
+func generateFingrprint(cmdName string, key string) string {
+	cmd := cmd.DiceDBCmd{
+		Cmd:  cmdName,
+		Args: []string{key},
+	}
+	return fmt.Sprintf("%d", cmd.GetFingerprint())
+}
+
```

Comment:
fingerprint generation logic can be hidden away from the test, we can simply assert the behavior using a static string.

Reply:
Understood. have done this change in the latest commit.